### ISSUE TITLE
Who's doing what?

### ISSUE THREAD

**Issue by drphilmarshall** [DarkEnergyScienceCollaboration/TDCcosmoforecasts#2]
Hi @mtewes - Apologies, it's been a while since we discussed the TDC cosmological forecasts project, not least because I've been busy preparing the ground for TDC2! But, I hope you are still interested in the forecasting exercise based on the TDC1 results. I made some progress on the CosmoSIS side of things with Dave Finley (@davidfinley) and Elise Jennings ( @EliseJ )  at the Argonne LSST DESC collaboration meeting back in the fall. Check out our notebook:

https://github.com/drphilmarshall/OM10/blob/master/examples/LSST/OM10_LSSTDESC.ipynb

Hopefully, Dave and Elise will continue to help out with CosmoSIS implementation and contribute to the cosmology discussion. (Sorry it took me so long to pass on our hack, you two!)  

Meanwhile, Tom Collett ( @tcollett ) has joined the DESC and is interested in this project, so perhaps together we can figure out who's doing what. Maybe a good place to start is to revisit [our rough plan, as written in the README](https://github.com/DarkEnergyScienceCollaboration/TDCcosmoforecasts/blob/master/README.md), and suggest changes and extensions? Tom, do you have questions about the project?

**Comment by mtewes** [comment]
Hello @drphilmarshall and all, my turn to present apologies for the slow reply... Yes, I'm still interested in this forecasting, and it's great to see this nice notebook!
I'll have a closer look at it (and at CosmoSIS, finally) this week. 

"TDC2" sounds like some work for good teams is ahead :)

**Comment by drphilmarshall** [comment]
Yeah! Should be fun - Kai is making an example dataset for you to look at
:-)

I think there's a problem with the current notebook, related to our
treatment of the lognormal model likelihood for D_dt given data (perhaps in
the normalization). The TDC2 emulated cosmological analysis is purposely
simpler, but we ought to be able to use the lognormal form in Suyu et al...
Something to look out for as you dive into CosmoSIS! I'll be looking at it
again next week, as I need to make some simple, non-TDC forecasts for a
review that Tommaso and I are writing - so between us we ought to be able
to figure out how it works! I think Elise will be happy to support us too
:-)

**Comment by EliseJ** [comment]
I'm happy to help out with this! I wouldn't be surprised if there are some bugs as this was a super quick hack.  The create_mocks module was not tested properly and there may be an issue with the lognormal rvs here. The first thing to check would be the mock data being output. 
If you send me a z_d,z_s file I can test this to make sure the Ddt_obs,err make sense.

**Comment by drphilmarshall** [comment]
Great, thanks Elise! All the files used in our example notebook (and the
notebook itself) are checked in to the OM10 module here:

https://github.com/drphilmarshall/OM10/tree/master/examples/LSST

Thank you! :-)
